Tensile/KernelWriter.py

```diff
@@ -96,45 +98,85 @@ def makeSchedule(self, kernel, tensorParametersA, tensorParametersB, localWriteE
       readCnt = self.globalReadACode.middle.countType(Code.GlobalReadInst) + \
                 self.globalReadBCode.middle.countType(Code.GlobalReadInst)
       # reads and incs are scheduled in iters range(0...endIter)
-      endIter = readCnt + 2 # 2 for incA and incB
+      # treat incA and incB as independent tasks that can be in the same iter code as Load(s) 
+      # 
+      numTaskstoSched += readCnt;
+      numTaskstoSched += 2 # 2 for incA and incB
```

Comment:
there are enable[] switches that can disable the increments (for profiling).  Should account for those here?